examples/rosa-classic-public-with-idp-machine-pools/main.tf

```diff
@@ -42,6 +42,64 @@ module "gitlab_idp" {
   gitlab_idp_url           = "https://gitlab.com"
 }
 
+module "htpasswd_idp" {
+  source = "../../modules/idp"
+
+  cluster_id         = module.rosa.cluster_id
+  name               = "htpasswd-idp"
+  idp_type           = "htpasswd"
+  htpasswd_idp_users = [{ username = "test-user", password = random_password.password.result }]
```

Comment:
I know it's just an example but should there be an output for that random password? Or there is no way to test log in to that user :thinking: